prisma/schema.prisma

```diff
@@ -47,7 +47,7 @@ model User {
   id            Int          @id @default(autoincrement())
   email         String?      @unique
   password      String?
-  name          String?
+  name          String?      @unique
```

Comment:
why do we need this to be unique? We find users by email primarily

Reply:
I thought we think of building a leader board or something ?

Reply:
Ask about this in the Discord to see what everyone thinks